pyccel/parser/semantic.py

```diff
@@ -1986,6 +2018,9 @@ def _ensure_inferred_type_matches_existing(self, class_type, d_var, var, is_auga
         rhs : TypedAstNode
             The right hand side of the expression : lhs=rhs.
             If is_augassign is False, this value is not used.
+        arr_in_multirets : bool, default: False
```

Comment:
Should we call this variable `mem_in_multirets` instead? That would be consistent with the method `is_memory_allocated_in_function`